README.md

```diff
@@ -48,29 +46,8 @@ packages with all of the major cloud native components.
 
 ### Postgres database
 
-For persistence storage, the project includes a Postgres database.
-
-On the VPS (Virtual Private Server) the DB runs within a docker container. For using Postgres from our local development environment, see the [Connecting to Postgres](#Connecting-to-Postgres) section.
-In order to establish connection from developer environment requires to create own docker Postgres container.
-Once the container runs, set up the development environment is required to set up credential within application.properties file.
-As the following:
-   1. Optional to set up an environment variable (dec env) for encryptor MASTER_PASSWORD, POSTGRES_PWD, POSTGRES_USR (as the VPS docker were set up)
-   2. Generate credential within ./starter dir, using the following command:
-   bash:   - mvn jasypt:encrypt-value -Djasypt.encryptor.password=$BONNEE_MASTER_PWD -Djasypt.encryptor.algorithm=PBEWithMD5AndDES -Djasypt.plugin.value=$POSTGRES_USR
-   bash:   - mvn jasypt:encrypt-value -Djasypt.encryptor.password=$BONNEE_MASTER_PWD -Djasypt.encryptor.algorithm=PBEWithMD5AndDES -Djasypt.plugin.value=$POSTGRES_PWD
-   3. Grab the output value of each, (it starts with ENC(..., ) then place into the application properties file
-   e.g: spring.datasource.password=ENC(6KpVjqrPwKvLt/5Cjo2ZHg==)
-
-Possible error:
-No jasypt.plugin.value property provided -> Add "..." around the parameters like: 
-    
-    "-Djasypt.encryptor.password=$BONNEE_MASTER_PWD" "-Djasypt.encryptor.algorithm=PBEWithMD5AndDES" "-Djasypt.plugin.value=$POSTGRES_USR"
-
-### VPS Docker Postgres setup for developer environment
-                                                     
-docker run -d --name "${USER}_postgres" -p 127.0.0.1:[pick free port num]:5432 -e POSTGRES_PASSWORD=doodle -e POSTGRES_USER=doodle -v /home/$USER/postgres/db:/var/lib/postgresql/data postgres
-
-ssh -N [${USER}]@bonnee.eu -L 5432:127.0.0.1:[picked port number]
+For persistence storage, the project includes a Postgres database running on the VPS (Virtual Private Server) within a docker container.
+To set up and connect to Postgres from your local development environment see the postgres documentation in docs folder.
```

Comment:
Could you please link the doc here? The navigation will be easier

Reply:
Yes, good point. Edited.